packages/esm-system-admin-app/src/root.component.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
 
-import { SystemAdministrationDashbord } from './dashboard/index.component';
+import { SystemAdministrationDashboard } from './dashboard/index.component';
 
 const RootComponent: React.FC = () => {
   return (
     <BrowserRouter basename={`${window.spaBase}/system-administration`}>
       <Routes>
-        <Route path="/" element={<SystemAdministrationDashbord />} />
+        <Route path="/" element={<SystemAdministrationDashboard />} />
```

Comment:
We don't actually need the path...

```suggestion
        <Route element={<SystemAdministrationDashboard />} />
```